uvicorn/protocols/http/httptools_impl.py

```diff
@@ -58,6 +58,14 @@ def __init__(
         self.access_logger = logging.getLogger("uvicorn.access")
         self.access_log = self.access_logger.hasHandlers()
         self.parser = httptools.HttpRequestParser(self)
+
+        try:
+            # Enable dangerous leniencies to allow server to a response on the first request from a pipelined request.
+            self.parser.set_dangerous_leniencies(lenient_data_after_close=True)
+        except AttributeError:
+            # httptools < 0.6.3
+            pass
```

Comment:
Some people don't use `uvicorn[standard]` because it install more than needed on production e.g. watchfiles.